sdk/eventhub/event-hubs/samples/useWithIotHub.ts

```diff
@@ -10,13 +10,13 @@ import { EventHubClient } from "@azure/event-hubs";
 const connectionString = "";
 
 async function main(): Promise<void> {
-  const client = await EventHubClient.createFromIotHubConnectionString(connectionString);
+  const client = new EventHubClient(connectionString);
```

Comment:
Add a comment above this that the connection string should be event hub compatible, and maybe provide pointers on how to get it from the portal